toolkits/global/packages/global-corporate-footer/view/corporate-level.hbs

```diff
@@ -0,0 +1,26 @@
+<div class="c-corporate-footer">
```

Comment:
this file should be called `corporate-footer`

Reply:
I guess you are right right. But then I believe when the package is enhance to include the product level part, that one will have to be called `c-product-footer` or something alike. Although in the same package they would be decoupled by class names... What do you think?

Reply:
if we have the class `c-corporate-footer` then we can use the filename `corporate-footer`
then if we have `c-product-footer` then we can use the filename `product-footer`
with possibly a `footer.hbs` that functions as the wrapper for both?

Reply:
It might be that once we have the product-specific stuff that `corporate-footer` is the wrapper (if we are keeping the component name) and then we have something like `_product-footer.hbs` and `_standard-footer.hbs`